sdk/formrecognizer/azure-ai-formrecognizer/CHANGELOG.md

```diff
@@ -1,7 +1,16 @@
 # Release History
 
 ## 1.0.0-beta.3 (Unreleased)
-- Added AAD authentication support
+- Update FormRecognizer API calls to return a `List` instead of `IterableStream`.
```

Comment:
In the future can you update the changelog in the PR you make the change?
i.e., if you have a PR to rename parameter `a` to `b`, can you put the changelog change in that PR? Thanks!

Reply:
I think I already did it? This PR should only include the Added support for AAD authentication.

Reply:
I was more referring to the other changes listed in the changelog for this PR, i.e. "Update FormRecognizer API calls to return a `List` instead of `IterableStream`." when they weren't changed in this PR

Reply:
Oh, It should be git merging issue. Now it has not showing the extra lines as new added line:
https://github.com/Azure/azure-sdk-for-java/pull/11156/files#diff-d6f6b064f65b639c74c8a74e6816a889R22